docs/guides/installation.md

```diff
@@ -24,10 +24,174 @@ Installation requires the following:
 Clone the [ZARP workflow repository][zarp] with:
 
 ```sh
-git clone git@github.com:zavolanlab/zarp
-# or: git clone https://github.com/zavolanlab/zarp.git
+git clone https://github.com/zavolanlab/zarp.git
 ```
 
 ### 2. Set up Conda environment
 
-**TODO:** Add on...
+To check if you already have conda installed in your system type:
```

Comment:
Let's try to be consistent with the names of tools. It's Conda, Mamba, Snakemake and ZARP (or _ZARP_) when referring to their names, and `conda`, `mamba`, `snakemake` and `zarp` (for ZARP-cli) when referring to the commands.

This is not followed consistently across many places, so I won't keep on commenting on it.

Reply:
I think I fixed it everywhere now